actions/create_uplink.py

```diff
@@ -15,7 +15,6 @@ def create_uplink(api_auth, parameters, contexts):
     :rtype: string
     """
     try:
-        site_type = parameters["SiteTypes"]
         city = parameters["City"]
```

Comment:
Should we also include the country just in case there are duplicate city names in different countries?
E.g. Sydney in Canada vs. Sydney in Australia

There's also Vinh's get_site_id_by_name() function that you can reuse if you want. You will need to pull it from master or something. Don't have to use it if you don't want to.

Reply:
Lol. Wrong line, but I think you get the point...